.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "TypeScriptToLua",
+    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:14",
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "extensions": [
+        "ark120202.vscode-typescript-to-lua",
```

Comment:
Why is this needed to develop tstl? Isn't this only for when you want to use it?

Reply:
We have 2 lua projects in this repo - lualib and benchmark